### ISSUE TITLE
[Bug] Noun declensions are missing

### ISSUE THREAD

**Issue by viktorprogger** [vuepress-theme-hope/vuepress-theme-hope#1924]
### Checklist

- [X] I am using the **latest version**
- [X] I have searched FAQ ([V1 Docs](https://vuepress-theme-hope.github.io/v1/FAQ/) | [V2 Docs](https://vuepress-theme-hope.github.io/v2/faq.html))

### Package name

vuepress-theme-hope

### Version

v2

### Which operating system are you using?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Describe the bug

In russian localization it's necessary for nouns to have declensions depending on bounden quantity. I.e.:
1 категория
3 категории
5 категорий

![image](https://user-images.githubusercontent.com/7670669/171015611-59e8f82f-6bba-4d58-a40a-1f60847e659e.png) ![image](https://user-images.githubusercontent.com/7670669/171015720-1339b94d-14a5-444d-8a26-ea5024010a88.png)

In English there are just 2 declensions: singular and plural. But "category" is always a singular, and "tags" is always a plural.
![image](https://user-images.githubusercontent.com/7670669/171016360-52a353c9-818a-4ebc-b99f-6f45e870a126.png) ![image](https://user-images.githubusercontent.com/7670669/171016381-95fd32f5-1755-44d4-8e07-94c43c6bea8d.png)

Here are just 2 cases, it's needed to be checked deeper.

**Comment by Mister-Hope** [comment]
You are free to improve i18n, just open new PRs.

**Comment by viktorprogger** [comment]
- Won't it be better to write "1 tag" and "2 tags"?
- Where can I find i18n-related code?

**Comment by Mister-Hope** [comment]
> - Won't it be better to write "1 tag" and "2 tags"?

Yes, but not all languages have multiple noun, e.g: Chinese doesn't 


> - Where can I find i18n-related code?

Search locale in the repo

**Comment by Mister-Hope** [comment]
Any update about this? Are you meeting trouble or just do not have time to fix it?

**Comment by Mister-Hope** [comment]
Closing as no response.

**Comment by viktorprogger** [comment]
Yes, I don't have time for it, sorry
